canairio/settings/base.py

```diff
@@ -16,23 +16,20 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 from datetime import timedelta
 
-ENVIRONMENT = 'development'
+def getenvvar(name):
+    v = os.environ.get(name)
+    if not v:
+        raise Exception('Environment variable {} undefined'.format(name))
```

Comment:
if the variable doesn't exist we should always be able to fallback to a default value, which is a feature provided by `getenv` and similar methods of retrieving environment variables.

Reply:
That's true, maybe we can do both here by passing an argument with a default value if needed and allowing the exception to be raised when we're not certain of a possible fallback.